nautobot/utilities/testing/views.py

```diff
@@ -491,7 +495,22 @@ def get_deletable_object(self):
             For some models this may just be any random object, but when we have FKs with `on_delete=models.PROTECT`
             (as is often the case) we need to find or create an instance that doesn't have such entanglements.
             """
-            return get_deletable_objects(self.model, self._get_queryset()).first()
+            instance = get_deletable_objects(self.model, self._get_queryset()).first()
+            if instance is None:
+                self.fail("Couldn't find a single deletable object!")
+            return instance
+
+        def get_deletable_object_pks(self):
+            """
+            Get a list of PKs corresponding to objects that can be safely bulk-deleted.
+
+            For some models this may just be any random objects, but when we have FKs with `on_delete=models.PROTECT`
+            (as is often the case) we need to find or create an instance that doesn't have such entanglements.
+            """
+            instances = get_deletable_objects(self.model, self._get_queryset()).values_list("pk", flat=True)[:3]
+            if len(instances) < 3:
+                self.fail(f"Couldn't find 3 deletable objects, only found {len(instances)}!")
+            return instances
```

Comment:
Is this the right place to be adding this? I'd thought that the `nautobot/utilities/testing/api.py` was the one we were having issues with. Do the `views.py` tests even need/use a `get_deletable_object_pks` function?

Reply:
I'm sorry I thought that was where you wanted it added. I'll go ahead and fix.